docs/src/pages/index.tsx

```diff
@@ -3,30 +3,55 @@ import clsx from 'clsx';
 import Layout from '@theme/Layout';
 import Link from '@docusaurus/Link';
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import GitHubButton from 'react-github-btn';
+import classnames from 'classnames';
+import useBaseUrl from '@docusaurus/useBaseUrl';
 
 import styles from './index.module.css';
 
-function HomepageHeader() {
-  const {siteConfig} = useDocusaurusContext();
+const Button = ({children, href}) => {
   return (
-    <header className={clsx('hero hero--primary', styles.heroBanner)}>
+    <Link className="button button--outline button--primary button--lg margin-horiz--sm" to={href}>
```

Comment:
Does the `className` need to be hardcoded? Can't we take it out of `styles` or `classnames`? Seems fragile 😁

Reply:
Not a deal breaker, we can keep it as-is – it should be possible but I was not able to make typescript happy quickly as well.